.github/workflows/release.yml

```diff
@@ -131,6 +133,10 @@ jobs:
         with:
           name: release-tars-${{env.PLATFORM_CLEAN}}
           path: src/github.com/containerd/containerd/releases/*.tar.gz*
+      - name: Attest Artifacts
```

Comment:
Can you move this into the `release` job?  We shouldn't be attesting binaries built as part of PRs.

Reply:
Yes. 
I assumed I could test in build job but now I figured out to run it in release in my fork.